matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -36,6 +36,37 @@ use uuid::Uuid;
 
 type Packet = Box<[u8]>;
 
+#[derive(Debug)]
+/// General configuration options for a WebRtc connection
+pub struct WebRtcSocketConfig {
+    pub room_url: String,
+    /// Configuration for the (single) ICE server
+    pub ice_server: RtcIceServerConfig
+}
+
+#[derive(Debug)]
+/// Configuration options for an ICE server connection.
+/// See also: https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer#example
+pub struct RtcIceServerConfig {
+    /// An ICE server instance can have several URLs. Use an empty Vec to disable ICE
```

Comment:
> Use an empty Vec to disable ICE

Are you sure this is allowed? MDN says it is required and should contain one or more urls.

If we want the ability to run without an ice server, I think it is the `RtcConfig.iceServers`/`ice_server_config_list` that should be empty, not the url list for the single server.

If we want a way to disable ICE, I think a better API would be to make `WebRtcSocketConfig.ice_server` an `Option<RtcIceServerConfig>` or ideally a `Vec<RtcIceServerConfig>` so we can support multiple servers as well (and it would mirror the underlying API).

Good idea to make it a struct, though. Makes it easier when/if we add TURN support.

Reply:
I see - to be honest I just tried it out and documented what I saw happening. Perhaps it only worked because I had a local-only setup. Probably it's best if I just remove that comment for now to avoid confusion

Reply:
I also like your suggestion and had similar ideas - I just wanted to keep this PR as minimal as possible and just add configurability without changing anything in the actual implementation because I don't know too much about the topic and dont't want to accidentally break something :)